tester_app/client/app/bundles/HelloWorld/constants/helloWorldConstants.jsx

```diff
@@ -0,0 +1,5 @@
+import mirrorCreator from 'mirror-creator';
```

Comment:
Maybe instead of copying code:

// See client/app/bundles/HelloWorld/actions/helloWorldActionCreators.jsx